docs/cli.md

```diff
@@ -74,6 +74,13 @@ By default, cucumber works in _strict_ mode, meaning it will fail if there are p
 
 See [Parallel](./parallel.md).
 
+## Printing Attachments Details
+
+Printing attachments details can be disabled with
```

Comment:
I think this content would sit better in `formatters.md` under the "Options" heading; this doc will shrink over time as we decouple config from the CLI.

Reply:
I have added some docs in `formatters.md`.
However I think keeping the one in `cli.md` could still be relevant?

Reply:
No problem!